password/password.go

```diff
@@ -0,0 +1,21 @@
+package password
+
+import (
+	"strings"
+
+	"github.com/dchest/uniuri"
+)
+
+// GeneratePassword generates a password for `secretName` if it doesn't already exist
+func GeneratePassword(secretData map[string][]byte, secretName string) bool {
+	secretKey := strings.Replace(strings.ToLower(secretName), "_", "-", -1)
```

Comment:
This transformation is performed 2 more times in `ssh/ssh.go` and will also be needed in `ssl/ssl.go`, so should be moved to a utility function somewhere.

Reply:
Fixed in 1babac7